src/core/libraries/VestingMath.sol

```diff
@@ -99,19 +99,21 @@ library VestingMath {
     /// @dev Lockup linear model uses the following distribution function:
     ///
     /// $$
-    /// f(x) = x * d + c
+    /// f(x) = x * sa + s + c
     /// $$
     ///
     /// Where:
     ///
-    /// - $x$ is the elapsed time divided by the stream's total duration.
-    /// - $d$ is the deposited amount.
-    /// - $c$ is the cliff amount.
+    /// - $x$ is the elapsed time in the streamable range divided by the streamable total duration.
+    /// - $sa$ is the streamable amount, i.e. deposited amount minus unlock amounts' sum.
```

Comment:
Since its a library, we can be explicit to avoid any confusion
```suggestion
    /// - $sa$ is the streamable amount, i.e. deposited amount - start unlocked amount - cliff unlocked amount.
```

Reply:
Fair enough.